src/pages/AddNewCake.js

```diff
@@ -12,6 +12,25 @@ export default function AddNewCake() {
     });
   };
 
+  const [image, setImage] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const uploadImage = async (event) => {
+    const files = event.target.files;
+    const data = new FormData();
+    data.append("file", files[0]);
+    data.append("upload_preset", "v6aqpvob");
+    setIsLoading(true);
+
+    const res = await fetch(
+      "https://api.cloudinary.com/v1_1/cakesapp/image/upload",
+      { method: "POST", body: data }
+    );
+    const file = await res.json();
+
+    setImage(file);
+  };
+
```

Comment:
Is this what's uploading the images? Nice! What is that strange named string in line 22? The id?

Reply:
This is taken from the upload settings from cloudinary. This is an interesting resource related to the topic 
https://medium.com/geekculture/how-to-upload-images-to-cloudinary-with-a-react-app-f0dcc357999c

and another one: 
https://cloudinary.com/documentation/image_upload_api_reference